<?xml version="1.0"?>
<launch>    
  <!-- Start the Dynamixel Joint States Publisher -->
  <node name="dynamixel_joint_states_publisher" pkg="dynamixel_joint_state_publisher" type="dynamixel_joint_state_publisher.py" output="screen">       
    <rosparam>
      joint_controllers: ['pan_controller', 'tilt_controller']
    </rosparam>
    <remap from="joint_states" to="pantilt_joint_states" />
  </node>
</launch>
